offload/test/offloading/CUDA/basic_launch_blocks_and_threads.cu

```diff
@@ -0,0 +1,30 @@
+// RUN: %clang++ -foffload-via-llvm --offload-arch=native %s -o %t
+// RUN: %t | %fcheck-generic
+
+// UNSUPPORTED: aarch64-unknown-linux-gnu
+// UNSUPPORTED: aarch64-unknown-linux-gnu-LTO
+// UNSUPPORTED: x86_64-pc-linux-gnu
+// UNSUPPORTED: x86_64-pc-linux-gnu-LTO
```

Comment:
Better to enable supported cases?

Reply:
Very true, but we should do that independently.
I'll open an issue or ask someone to look into it.